tests/components/sonos/test_media_player.py

```diff
@@ -1337,3 +1362,210 @@ async def test_service_update_alarm_dne(
             blocking=True,
         )
     assert soco.alarmClock.UpdateAlarm.call_count == 0
+
+
+async def test_position_updates(
+    hass: HomeAssistant,
+    soco: MockSoCo,
+    async_autosetup_sonos,
+    media_event: SonosMockEvent,
+    current_track_info: dict[str, Any],
+) -> None:
+    """Test the media player position updates."""
+
+    freezetime = datetime(2024, 1, 1, 12, 0, 0, 0, tzinfo=UTC)
+    with freeze_time(freezetime) as frozen_time:
+        soco.get_current_track_info.return_value = current_track_info
```

Comment:
You can use a parameter for this right? `@pytest.mark.freeze_time("2023-10-21")`

Reply:
Indeed - updated